### DISCUSSION TITLE
how to write a distributed application in kubernetes

### DISCUSSION THREAD

**Discussion by SamaelChen** [hpcaitech/ColossalAI#3214] [Community | Q&A]
Any documents about how to initialize distributed environment in kubernetes?

**Comment by kurisusnowdeng** [comment]
Hi @SamaelChen , coincidentally we are now testing ColossalAI with AWS EKS. In a nutshell, you could launch nodes with the docker image pulled from [our docker hub](https://hub.docker.com/r/hpcaitech/colossalai/tags) (or an image that includes ColossalAI built by yourself), and then run your task in the same way as in other multi-node/GPU distributed clusters. You are also very welcome to share any related experience with us!
